syntax = "proto3";

package payment_service;

service Payment_Service {
    rpc Request_Commit (Request_Commit_Message) returns (Response);
    rpc Commit (Commit_Message) returns (Response);
}
message Response {
    bool status = 1; 
    string message = 2; 
}
message Request_Commit_Message{
    int32 id = 1;
    
}
message Commit_Message{
    int32 id = 1;
    bool rollback = 2; 
}